test/mpi/Makefile.mtest

```diff
@@ -16,20 +16,18 @@
 
 # AM_CPPFLAGS are used for C++ code as well
 AM_CPPFLAGS = -I$(top_builddir)/include -I$(top_srcdir)/include
-LDADD = $(top_builddir)/util/mtest.o
+LDADD = $(top_builddir)/util/mtest.$(OBJEXT) $(top_builddir)/util/mtest_common.$(OBJEXT)
```

Comment:
Changing `mtest.o` to `mtest.$(OBJEXT)` is a bug-fix and should be separated into a different commit.